src/Pages/SignUp/SignUp.js

```diff
@@ -1,8 +1,137 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 import './SignUp.scss';
 
 export default class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      pwd: '',
+      name: '',
+    };
+  }
+
+  clickHandler = () => {
+    //e.prevent.Default();
+    fetch('http://10.58.0.88:8000/user/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.pwd,
+        name: this.state.name,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res.message === 'SUCCESS') {
+          this.props.history.push('/login');
+        }
+      });
+  };
+
+  //패스워드 유효성 검사
+  checkPwdHandler = (e) => {
+    e.preventDefault();
+    const checkPwd = function (str) {
+      var reg_pwd = /^.*(?=.{6,20})(?=.*[0-9])(?=.*[a-zA-Z]).*$/;
+      return !reg_pwd.test(str) ? false : true;
+    };
+    if (checkPwd(this.state.pwd) === false) {
```

Comment:
```suggestion
    if (checkPwd(this.state.pwd)) {
```

불리언 값을 리턴하는 함수라면 이렇게만 써도 if문이 동작할거 같네요. 반대값을 표현하시려면 !(부정연산자)를 앞에 붙여주시면 되는거 아시죠?

Reply:
바로 고쳤습니다 감사합니다!